src/models/testSuite.ts

```diff
@@ -47,6 +58,11 @@ export type TestSuiteWithExecution = {
   latestExecution?: TestSuiteExecution;
 };
 
+export type TestSuiteWithExecutionRedux = {
```

Comment:
Same as above, may be good to align redux and regular version in follow up task (probably even the same as above)